recipes/cpp_project_framework/all/test_package/CMakeLists.txt

```
@@ -1,14 +1,15 @@
 cmake_minimum_required(VERSION 3.1)
 project(PackageTest CXX)
 
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
+
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
 conan_basic_setup()
 
 add_executable(test_package test_package.cpp)
-target_link_libraries(test_package ${CONAN_LIBS})
-
-# CTest is a testing tool that can be used to test your project.
-enable_testing()
-add_test(NAME test_package
-         WORKING_DIRECTORY ${CMAKE_BINARY_DIR}/bin
-         COMMAND test_package)
+if(UNIX)
+	target_link_libraries(test_package ${CONAN_LIBS} stdc++fs)
```

its marked header only so it should not need to link to libs?

```suggestion
	target_link_libraries(test_package stdc++fs)
```

---

the test depends on `gtest` lib

---

Ahh thank you... the diff was hiding.

This was is deprecated and we are trying to not add it https://github.com/conan-io/conan-center-index/blob/master/docs/reviewing.md#cmake-targets

---

changed in latest version